sql-statements/sql-statement-create-index.md

```diff
@@ -104,44 +104,108 @@ Query OK, 0 rows affected (0.31 sec)
 
 ## Expression index
 
+In some scenarios, query conditions run a filtering process based on a certain expression. In these scenarios, the query performance is relatively poor because ordinary indexes cannot take effect, the query can only be executed by scanning the entire table. The expression index is a type of special index that can be built on an expression. Once an expression index is built, TiDB can use the index for the expression-based query, which significantly improves the query performance.
+
+For example, if you want to build an index based on `col1+cols2`, execute the following SQL statement:
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE INDEX idx1 ON t1 ((col1 + col2));
+```
+
+Or you can execute the following equivalent statement:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE t1 ADD INDEX idx1((col1 + col2));
+```
+
+You can also specify the expression index when you build the table:
+
+{{< copyable "sql" >}}
+
+```sql
+CREATE TABLE t1(col1 char(10), col2 char(10), key index((col1 + col2)));
+```
+
+You can drop an expression index in the same way as dropping an ordinary index:
+
+{{< copyable "sql" >}}
+
+```sql
+DROP INDEX idx1 ON t1;
+```
+
 > **Note:**
 >
-> Expression index is still an experimental feature. It is **NOT** recommended that you use it in the production environment.
+> An expression index cannot be created on a primary key.
+>
+> The expression in an expression index cannot contain the following content:
+>
+> - Volatile functions, such as `rand()`, `now()`, and so on.
+> - system variables and user variables.
+> -Subqueries.
+> - `AUTO_INCREMENT` column. However, there is one exception: you can remove this restriction by setting the value of `tidb_enable_auto_increment_in_generated` (system variable) to `true`.
+> - window functions.
+> - ROW functions, such as `create table t (j json, key k (((j,j))));`.
+> - aggregate functions.
```

Comment:
@en-jin19 Could you please ensure you haven't missed a global comment? https://github.com/pingcap/docs/pull/6185#discussion_r691908185

Reply:
```suggestion
> - Window functions.
> - ROW functions, such as `create table t (j json, key k (((j,j))));`.
> - Aggregate functions.
```